Kiss.Bff/Extern/InterneTaak/InterneTaakProxyConfig.cs

```diff
@@ -9,17 +9,18 @@ namespace Kiss.Bff.InterneTaak
 {
     public static class InterneTaakExtensions
     {
-        public static IServiceCollection AddInterneTaakProxy(this IServiceCollection services, string destination, string token, string objectTypeUrl, string clientId, string clientSecret)
-            => services.AddSingleton(new InterneTaakProxyConfig(destination, token, objectTypeUrl, clientId, clientSecret))
+        public static IServiceCollection AddInterneTaakProxy(this IServiceCollection services, string destination, string token, string objectTypeUrl, string clientId, string clientSecret, string typeVersion)
+            => services.AddSingleton(new InterneTaakProxyConfig(destination, token, objectTypeUrl, clientId, clientSecret, typeVersion))
             .AddSingleton<IKissProxyRoute>(s => s.GetRequiredService<InterneTaakProxyConfig>());
     }
 
     public class InterneTaakProxyConfig : IKissProxyRoute
     {
-        public InterneTaakProxyConfig(string destination, string token, string objectTypeUrl, string clientId, string clientSecret)
+        public InterneTaakProxyConfig(string destination, string token, string objectTypeUrl, string clientId, string clientSecret, string typeVersion)
         {
             Destination = destination;
             ObjectTypeUrl = objectTypeUrl;
+            TypeVersion = typeVersion;
```

Comment:
ziet er goed uit. het lijkt me wel een goed idee om typeversion een default waarde (1)  te geven.
en svp dit nieuwe configuratie element opnemen in de documentatie: https://github.com/Klantinteractie-Servicesysteem/.github/pulls